client/src/components/generic/ProfileInformationPanel/ProfileInformationPanel.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReactNode } from "react"
+import editButton from "assets/icons/edit.svg"
+interface IProfileInformationPanel {
+  title: string // The title field
+  children?: ReactNode
+  onEdit?: () => void // The edit button
+}
+
+const ProfileInformationPanel = ({
+  title,
+  children,
+  onEdit
+}: IProfileInformationPanel) => {
+  return (
+    // <div className="box-border flex w-[75%] flex-col border p-4">
+    // <div className="absolute max-w-[50%] flex-row rounded border border-stone-300 bg-white p-4">
+    <div className="absolute max-w-[90%] flex-row rounded border border-stone-300 bg-white p-4">
+      <div className="flex flex-nowrap justify-between text-nowrap p-1">
+        <h3 className="text-dark-blue-100">{title}</h3>
+        {onEdit && (
+          <img
+            className="flex-end max-w-[80%] cursor-pointer items-end pl-4 pr-4"
+            src={editButton}
+            alt="Edit"
+            onClick={onEdit}
+          />
```

Comment:
This isn't a preferred way to use svgs, checkout `Navbar.tsx` to see how the logo was used (normally we want to wrap them in another element to constrain size)

Reply:
![image](https://github.com/UoaWDCC/uasc-web/assets/100653148/edfd6242-b964-42d7-bd54-41d2039844d1)
Here we can see it growing